src/libServer/JSONConversion.cpp

```diff
@@ -39,46 +39,17 @@ using namespace boost::multiprecision;
 
 unsigned int JSON_TRAN_OBJECT_SIZE = 10;
 
-const Json::Value JSONConversion::convertBoolArraytoJson(
-    const vector<bool>& v) {
-  Json::Value jsonBool;
+const Json::Value JSONConversion::convertMicroBlockInfoArraytoJson(
+    const vector<MicroBlockInfo>& v) {
+  Json::Value mbInfosJson;
```

Comment:
I think you should initialize this with empty JsonArray. 
https://stackoverflow.com/questions/13293043/create-empty-json-array-with-jsoncpp